aptos-move/framework/aptos-framework/sources/ResourceAccount.move

```diff
@@ -0,0 +1,128 @@
+/// A resource account is used to manage resources independent of an account managed by a user.
+/// This contains several utilities to make using resource accounts more effective.
+///
+/// A dev wishing to use resource accounts for a liquidity pool, would likely do the following:
+/// 1. Create a new account using `ResourceAccount::create_resource_account`. This creates the
+/// account, stores the `signer_cap` within a `ResourceAccount::Container`, and rotates the key to
+/// the current accounts authentication key or a provided authentication key.
+/// 2. Define the LiquidityPool module's address to be the same as the resource account.
```

Comment:
how to publish liquidity pool's module to the resource account address?

Reply:
you own the authentication key, so you can publish modules to that address via a normal transaction. let me add more text here.